_posts/research/2023-03-24-hierarchical-ad-hoc-agents.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "HAHA: Hierarchical Ad Hoc Agents"
+description: Using hierarchical structures to improve human-agent interaction
+author: Stéphane Aroca-Ouellette
+permalink: research/haha.html
+category: [research, learning and modeling, haha, research highlight]
+image:
+    feature: research/haha/haha_architecture.png
+    size: 50%
+excerpt_separator: <!-- More -->
+---
+
+When performing collaborative tasks with new unknown teammates, humans are particularly adept at adapting to their collaborator 
+and converging toward an aligned strategy. However, state of the art autonomous agents still do not have this capability. 
+We propose that a critical reason for this disconnect is that there is an inherent hierarchical structure to human behavior 
+that current agents lack. In this research, we explore the use of hierarchical reinforcement learning to train an agent that can 
+navigate the complexities of ad hoc teaming at the same level of abstraction as humans. Our results demonstrate that when paired with humans, 
+our Hierarchical Ad Hoc Agent (HAHA) outperforms all baselines on both the team's objective performance and the human’s perception of the agent.  
+<!-- More -->
+
+# Published Work:
+## HAHA: Hierarchical Ad Hoc Agents
```

Comment:
This is unclear: are you going to have one page for all your pubs on the topic? If so, either change the page title or avoid duplicating it here.